test_data/yast/msdos/msdos.yaml

```diff
@@ -0,0 +1,46 @@
+table_type: msdos
+partition_table_disk: vda
+disks:
+  - name: vda
+    allowed_unpartitioned: 0.00GB
+    partitions:
+      - size: 8Mib
+        role: raw-volume
+        partition_type: primary
+        validation_flag: 0
+        formatting_options:
+          should_format: 1
+          filesystem: fat
+        mounting_options:
+          should_mount: 1
+          mount_point: '/boot/efi'
+      - size: 9250Mib
+        role: operating-system
+        partition_type: primary
+        validation_flag: 1
+        formatting_options:
+          should_format: 1
+          filesystem: xfs
+        mounting_options:
+          should_mount: 1
+          mount_point: '/'
+      - size: 9250Mib
+        role: data
+        partition_type: primary
+        validation_flag: 1
+        formatting_options:
+          should_format: 1
+          filesystem: xfs
+        mounting_options:
+          should_mount: 1
+          mount_point: '/home'
+      - size: max 
+        role: swap
+        partition_type: primary
+        validation_flag: 0 
+        formatting_options:
+          should_format: 1
+          filesystem: swap
+        mounting_options:
+          should_mount: 1
+          mount_point: '[SWAP]' 
```

Comment:
that may break when using function `select_filesystem`

Reply:
I tried this. At least on the the tests I run locally, it works

Reply:
I think we would type [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/d28a53d3057c7c7256932d92721ea473664ea228/lib/Installation/Partitioner/FormattingOptionsPage.pm#L125) as [SWAP] and should be visible in the screen, right? Let me check the VR, in the logs we could see what test data was used as well.

Reply:
Yes, it does fill it with '[SWAP]' but doesn't cause issue
http://falafel.suse.cz/tests/155#step/msdos_partition_table/84

Reply:
At least on this build xD

Reply:
I confirmed with YaST team that they might be sanitizing the input somehow. I was surprised that worked, they say that might even work with |SWAP|. Did you try with some other random string? maybe there is a bug :)
But yes, use that one, good discovery :)